backend/manager/db.py

```diff
@@ -0,0 +1,14 @@
+from config import Config
+import redis
+
+class DBManager:
+    __redis_store = None
+
+    @classmethod
+    def __init_redis(cls):
+        cls.__redis_store = redis.StrictRedis(host=Config.redis["host"], port=Config.redis["port"], db=0, charset="utf-8", decode_responses=True)
```

Comment:
this line looks too long https://google.github.io/styleguide/pyguide.html?showone=Line_length#Line_length